MapboxNavigationTests/NavigationMapViewTests.swift

```diff
@@ -52,4 +89,23 @@ class NavigationMapViewTests: XCTestCase {
         XCTAssertEqual(congestionSegmentsSevere[1].1, .severe)
         XCTAssertEqual(congestionSegmentsSevere[2].1, .low)
     }
+    
+    func testAnnotations() {
```

Comment:
This looks great. My main feedback would be to use a more descriptive test method name, for example `testRemoveWaypointsDoesNotRemoveUserAnnotations()` or some such.

Also, the route used for testing appears to have two waypoints (line 16, above). Is it expected that showing waypoints for the route would only add one waypoint to the map?

Reply:
> Is it expected that showing waypoints for the route would only add one waypoint to the map

Yes, the first one typically represents the user’s current location which we don't show as an annotation.

Reply:
got it, thanks